server/database/queries.js

```diff
@@ -102,8 +100,24 @@ const getListsAndCardsForBoard = (board) => {
         .orderBy('list_id', 'asc')
         .orderBy('order', 'asc')
         .then(cards => {
-          board.cards = cards
-          return board
+          const cardIds = cards.map( card => card.id )
+          return knex.table('card_labels')
+            .select('*')
+            .whereIn( 'card_id', cardIds )
+            .join('labels','card_labels.label_id','labels.id')
+            .then( cardLabels => {
+              cards = cards.map( card => {
+                card.labels = []
+                cardLabels.forEach( cardLabel => {
```

Comment:
I would not put copies of each label on each card. Store labels as a property on the board object and give each card an array of label id. this way we do not duplicate data and our data structure will be smaller